src/classes/BDI_DataImport_API.cls

```diff
@@ -42,83 +42,69 @@ global with sharing class BDI_DataImport_API {
     * @description string values for different matching behaviors BDI can support.  couldn't use an enum
     * since we need to store it in a custom settings text field.
     */
-    global static final string DoNotMatch = 'DoNotMatch'; // Don't Match. Always create new Opportunities.
-    global static final string RequireNoMatch = 'RequireNoMatch'; // Always create.  Show error if any match.
-    global static final string RequireExactMatch = 'RequireExactMatch'; // Use exact match. Show error if there isn't an exact match or there are no matches.
-    global static final string ExactMatchOrCreate = 'ExactMatchOrCreate'; // Use exact match. Show error if there isn't an exact match. Create opportunity if there are no matches.
-    global static final string RequireBestMatch = 'RequireBestMatch'; // Use the best match. Show error if no match.
-    global static final string BestMatchOrCreate = 'BestMatchOrCreate'; // Use the best match. Create opportunity if no match.
+    global static final string DoNotMatch = 'Do Not Match'; // Don't Match. Always create new Opportunities.
```

Comment:
I needed to change the values since they are now also used in the batch picklist.  Thus I wanted the picklist values to use the same wording that our expanded help text values (that we will show in the UI) are using.  What will happen if any settings stored the old values is that it won't show a choice in the UI, and when they run the batch, they will get an error that the choice isn't valid.  then they can easily fix it.  Since I'm not convinced many customers have even used the new donation support, the amount of people hitting this will be quite small.